docs/installing.rst

```diff
@@ -11,25 +11,27 @@ Installing the Atomic Developer Bundle
 1. Install a Virtualization Provider
```

Comment:
Hi Budh,
In this case they are sequential steps hence numbered. Have you found this issue in any other place too? I will look into this.